common/utils/src/main/resources/error/error-classes.json

```diff
@@ -92,6 +92,13 @@
       "The method <methodName> can not be called on streaming Dataset/DataFrame."
     ]
   },
+  "CANNOT_ALTER_PARTITION_COLUMN" : {
+    "message" : [
+      "ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns in non-delta tables.",
```

Comment:
Is it necessary to mention `non-delta` here. What does this mean for Spark users?

Reply:
You're right. We shouldn't specify third party formats. I'll change the error message to `ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns.`